hw/drivers/nimble/nrf52/src/ble_phy.c

```diff
@@ -1694,8 +1698,13 @@ ble_phy_setchan(uint8_t chan, uint32_t access_addr, uint32_t crcinit)
     /* Set the frequency and the data whitening initial value */
     g_ble_phy_data.phy_chan = chan;
     NRF_RADIO->FREQUENCY = freq;
+#if MYNEWT_VAL(BLE_LL_DIRECT_TEST_MODE) == 1
```

Comment:
this is redundant - you can set IV for data whitening regardless of DTM since it is disabled for DTM anyway (so IV does not matter)

Reply:
let me try that then. If OK I will remove it.